fsw/mission_inc/sample_platform_cfg.h

```diff
@@ -0,0 +1,29 @@
+/************************************************************************
```

Comment:
@mgrubb-gsfc  - This file should be in the `platform_inc` directory

Reply:
Resolved in commit https://github.com/nasa-itc/sample/pull/1/commits/78c4887258bf031e6dd325db3b81f0502ad82d7f